test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -100,6 +100,82 @@ def test_capture_server(self):
+        all_daemons_up = all_daemons_up and self.scheduler.isUp()
+
+        if all_daemons_up is False:
+            # Skip the test
+            self.skipTest("Type 1 installation not present or " +
+                          "all daemons are not running")
+
+        with PBSSnapUtils(out_dir=output_dir, acct_logs=num_acct_logs,
+                          srvc_logs=num_service_logs, sudo=True) as snap_obj:
+            snap_obj.capture_all()
+            snap_obj.finalize()
+
+            # Test that all the expected information has been captured
+            # PBSSnapUtils has various dictionaries which store metadata
+            # for various objects. Create a list of these dicts
+            all_info = [snap_obj.server_info, snap_obj.job_info,
+                        snap_obj.node_info, snap_obj.comm_info,
+                        snap_obj.queue_info, snap_obj.hook_info,
+                        snap_obj.sched_info, snap_obj.resv_info,
+                        snap_obj.rscs_info, snap_obj.datastore_info,
+                        snap_obj.pbs_info, snap_obj.core_info,
+                        snap_obj.sys_info, snap_obj.misc_info]
+            skip_list = [snap_obj.ACCT_LOGS, snap_obj.QMGR_LPBSHOOK_OUT,
+                         "reservation", "job", snap_obj.QMGR_PR_OUT,
+                         snap_obj.PG_LOGS, "core_file_bt",
+                         snap_obj.PBS_SNAPSHOT_LOG]
+            platform = self.du.get_platform()
+            if platform.startswith("linux") is False:
+                skip_list.extend([snap_obj.ETC_HOSTS,
+                                  snap_obj.ETC_NSSWITCH_CONF,
+                                  snap_obj.LSOF_PBS_OUT, snap_obj.VMSTAT_OUT,
+                                  snap_obj.DF_H_OUT, snap_obj.DMESG_OUT])
+            for item_info in all_info:
+                for key, info in item_info.iteritems():
+                    info_path = info[0]
+                    if info_path is None:
+                        continue
+                    # Check if we should skip checking this info
+                    skip_item = False
+                    for item in skip_list:
+                        if type(item) == int:
+                            if item == key:
+                                skip_item = True
+                                break
+                        else:
+                            if item in info_path:
+                                skip_item = True
+                                break
+                    if skip_item is True:
+                        continue
+
+                    # Check if this information was captured
+                    info_full_path = os.path.join(snap_obj.outdir,
+                                                  info_path)
+                    try:
+                        self.assertTrue(os.path.exists(info_full_path) is True)
+                    except:
+                        raise Exception(info_full_path + " was not captured")
```

Comment:
self.assertTrue() takes a message to be printed on failure.  Use that instead of the try/except block